recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc

```diff
@@ -304,7 +304,10 @@ FILES_libegl-imx-dev = "${includedir}/EGL ${includedir}/KHR ${libdir}/pkgconfig/
 
 FILES_libgal-imx = "${libdir}/libGAL${SOLIBS} ${libdir}/libGAL_egl${SOLIBS}"
 FILES_libgal-imx-dev = "${libdir}/libGAL${SOLIBSDEV} ${includedir}/HAL"
-RDEPENDS_libgal-imx += "kernel-module-imx-gpu-viv"
+RDEPENDS_libgal-imx += " \
+    kernel-module-imx-gpu-viv \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'wayland', 'valgrind', '', d)} \
```

Comment:
It is binded to `wayland` backend?

Reply:
It's what I was told. I'll ask again to make sure.